docs/resources/google_compute_firewalls.md.erb

```diff
@@ -0,0 +1,80 @@
+---
+title: About the google_compute_firewalls Resource
+platform: gcp
+---
+
+# google\_compute\_firewalls
+
+Use the `google_compute_firewalls` InSpec audit resource to test properties of all GCP compute firewalls for a project.
+
+<br>
+
+## Syntax
+
+A `google_compute_firewalls` resource block collects GCP firewalls by project then tests that group.
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      it { should exist }
+    end
+
+Use this InSpec resource to enumerate IDs then test in-depth using `google_compute_firewall`.
+
+    google_compute_firewalls(project: 'chef-inspec-gcp').firewall_names.each do |firewall_name|
+      describe google_compute_firewall(project: 'chef-inspec-gcp',  name: firewall_name) do
+        it { should exist }
+        its('kind') { should eq "compute#firewall" }
+      end
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test that there are no more than a specified number of firewalls available for the project
+
+    describe google_compute_firewalls(project: 'chef-inspec-gcp') do
+      its('entries.count') { should be <= 100}
```

Comment:
You can just use `its('count') ...` here.

Reply:
sure, will update